app/src/main/java/org/schabi/newpipe/player/ServicePlayerActivity.java

```diff
@@ -129,6 +128,7 @@ public boolean onCreateOptionsMenu(final Menu m) {
         getMenuInflater().inflate(R.menu.menu_play_queue, m);
         getMenuInflater().inflate(getPlayerOptionMenuResource(), m);
         onMaybeMuteChanged();
+        onPlaybackParameterChanged(player.getPlaybackParameters());
```

Comment:
Where is this coming from?

Reply:
This fixes the issue with the speed on the button being wrong